.jshintrc

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
please remove jshint and replace with eshint, see periodic's baseline eshintrc file for an example. https://github.com/typesettin/periodicjs/blob/master/.eslintrc.json